<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IterativeReconstructionInitializationParamsControlWidget</class> 
 <widget class="QWidget" name="IterativeReconstructionInitializationParamsControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IterativeReconstructionInitializationParams</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_0">
       <property name="text">
         <string>b2bRansacP5RPThreshold</string>
       </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="b2bRansacP5RPThresholdSpinBox">
       <property name="toolTip">
         <string>Best-2nd best essential estimator threshold</string>
       </property>
       <property name="minimum">
        <double>0</double>
       </property>
       <property name="maximum">
        <double>1</double>
       </property>
       <property name="singleStep">
        <double>1</double>
       </property>
       <property name="value">
        <double>0.8</double>
       </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_1">
       <property name="text">
         <string>inlierP5RPThreshold</string>
       </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="inlierP5RPThresholdSpinBox">
       <property name="toolTip">
         <string>Inlier threshold</string>
       </property>
       <property name="minimum">
        <double>0</double>
       </property>
       <property name="maximum">
        <double>50000</double>
       </property>
       <property name="singleStep">
        <double>1</double>
       </property>
       <property name="value">
        <double>5</double>
       </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
       <property name="text">
         <string>maxEssentialRansacIterations</string>
       </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="maxEssentialRansacIterationsSpinBox">
       <property name="toolTip">
         <string>Maximal essential estimator rounds</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>5000000</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
       <property name="value">
        <number>600000</number>
       </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
       <property name="text">
         <string>b2bRansacP6RPThreshold</string>
       </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="b2bRansacP6RPThresholdSpinBox">
       <property name="toolTip">
         <string>Best-2nd best relative pose estimator threshold</string>
       </property>
       <property name="minimum">
        <double>0</double>
       </property>
       <property name="maximum">
        <double>1</double>
       </property>
       <property name="singleStep">
        <double>1</double>
       </property>
       <property name="value">
        <double>0.8</double>
       </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
       <property name="text">
         <string>runEssentialFiltering</string>
       </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="runEssentialFilteringCheckBox">
       <property name="toolTip">
         <string>Run essential filtering prior relative pose estimation</string>
       </property>
       <property name="checked">
           <bool>1</bool>
       </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
       <property name="text">
         <string>essentialTargetGamma</string>
       </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QDoubleSpinBox" name="essentialTargetGammaSpinBox">
       <property name="toolTip">
         <string>essentialTargetGamma</string>
       </property>
       <property name="minimum">
        <double>0</double>
       </property>
       <property name="maximum">
        <double>1</double>
       </property>
       <property name="singleStep">
        <double>0.01</double>
       </property>
       <property name="value">
        <double>0.01</double>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_6">
       <property name="text">
         <string>maxP6RPIterations</string>
       </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QSpinBox" name="maxP6RPIterationsSpinBox">
       <property name="toolTip">
         <string>Maximal number of ransac iterations for pose estimation using pairwise correspondences</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>100000000</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>400000</number>
       </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_7">
       <property name="text">
         <string>inlierP6RPThreshold</string>
       </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QDoubleSpinBox" name="inlierP6RPThresholdSpinBox">
       <property name="toolTip">
         <string>Inlier threshold for pose estimation using pairwise correspondences</string>
       </property>
       <property name="minimum">
        <double>0</double>
       </property>
       <property name="maximum">
        <double>1000</double>
       </property>
       <property name="singleStep">
        <double>0.01</double>
       </property>
       <property name="value">
        <double>1</double>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_8">
       <property name="text">
         <string>gammaP6RP</string>
       </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QDoubleSpinBox" name="gammaP6RPSpinBox">
       <property name="toolTip">
         <string>Target failure probability for pose estimation</string>
       </property>
       <property name="minimum">
        <double>0</double>
       </property>
       <property name="maximum">
        <double>1</double>
       </property>
       <property name="singleStep">
        <double>0.01</double>
       </property>
       <property name="value">
        <double>0.01</double>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
     </widget>
    </item>
   <item row="9" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="3" margin="3"/>
 <resources/>
 <connections>
  <connection>
   <sender>b2bRansacP5RPThresholdSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>IterativeReconstructionInitializationParamsControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>inlierP5RPThresholdSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>IterativeReconstructionInitializationParamsControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>maxEssentialRansacIterationsSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>IterativeReconstructionInitializationParamsControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>b2bRansacP6RPThresholdSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>IterativeReconstructionInitializationParamsControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>runEssentialFilteringCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>IterativeReconstructionInitializationParamsControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>essentialTargetGammaSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>IterativeReconstructionInitializationParamsControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>maxP6RPIterationsSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>IterativeReconstructionInitializationParamsControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>inlierP6RPThresholdSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>IterativeReconstructionInitializationParamsControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>gammaP6RPSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>IterativeReconstructionInitializationParamsControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
 </connections>
 <slots>
  <slot>changeParameters()</slot>
 </slots>
 <customwidgets>
 </customwidgets>
</ui>
